Singleclick show hide label (#5485)
qgis/QGIS-Documentation: docs/user_manual/working_with_vector/vector_properties.rst

The :guilabel:`Label Toolbar` provides some tools to manipulate |labeling|
:ref:`label <vector_labels_tab>` or |diagram| :ref:`diagram <sec_diagram>`

> properties but only if the corresponding data-defined option is indicated.

Reviewer: ```suggestion
properties, but only if the corresponding data-defined option is indicated.
```

* |pinLabels| :sup:`Pin/Unpin Labels And Diagrams` that has data-defined
  position. By clicking or draging an area, you pin label(s). If you click or

>   drag an area holding :kbd:`Shift` label(s) are unpinned. Finally, you can

Reviewer: ```suggestion
  drag an area holding :kbd:`Shift`, label(s) are unpinned. Finally, you can
```

  determine whether any important labels are missing from the maps (e.g. due
  to overlaps or other constraints). They are displayed with a customizable
  color (see :ref:`automated_placement`).
* |showHideLabels| :sup:`Show/Hide Labels And Diagrams`. If you click on the labels,

>   then they are hidden. When a label is hidden, you just have to click

Reviewer: ```suggestion
  or click and drag an area holding :kbd:`Shift`, they are hidden.
  When a label is hidden, you just have to click
```

>   on the feature to restore its visibility. This tool also works without setting

Reviewer: ```suggestion
  on the feature to restore its visibility.
  If you drag an area, all the labels in the area will be restored.
  This tool also works without setting
```

  data-defined options.
* |moveLabel| :sup:`Move Label And Diagram` that has data-defined
  position. You just have to drag the label to the desired place. This only
  works when you set the data-defined override for the :guilabel:`Data defined`

>   entries.

Reviewer: I am not sure if I understand this. Does "Move Label And Diagram" only work for data-defined positions?
I was able to use the tool to move labels that were "Single Labels", without having anything "data-defined".
Perhaps "data-defined" here means something else (that the location of labels is determined from feature positions, and not static?).

Author: @hatav I checked the Label Toolbar: all tools work fine without setting the data-defined coordinates
